docs/source/resources/common/exposing.md

```diff
@@ -1,77 +1,124 @@
-# Exposing ScyllaCluster
+# Exposing ScyllaDB cluster
 
-This document explains how ScyllaDB Operator exposes ScyllaClusters in different network setups.
-A ScyllaCluster can be exposed in various network configurations, independently to clients and nodes.
-
-:::{note}
-ScyllaClusters can be only exposed when the ScyllaDB version used version is `>=2023.1` ScyllaDB Enterprise or `>=5.2` ScyllaDB Open Source.
-:::
+This document explains how ScyllaDB Operator exposes ScyllaDB clusters in different network setups.
+A ScyllaDB Cluster can be exposed in various network configurations, independently to clients and nodes.
```

Comment:
nit: s/Cluster/cluster (for consistency)

Reply:
done